### PR TITLE
Updated and synced feature lists and updated redirect links

### PR DESCRIPTION
**Pull request by UtiluMark**
Updated and synced the feature lists in the README and the About window.
Updated the AMO links in the README and the About window to prevent
redirects.
Added link to Screengrab (fix version) add-on.
Part of #188

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -58,6 +58,6 @@
   * Repository: Reorganize repository and documentation (#28)
   * Extension: Update list of contributors (#38)
   * Extension: Update maxVersions for applications (#37)
-  * Extension: Allow title bar customization in Thunderbird (#24)
+  * Extension: Allow titlebar customization in Thunderbird (#24)
```
Comment (whimboo) [inline History.md]: The former wording was correct. So please revert this change.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-* Build ID retrieval
-* Copy extension list to clipboard
-* Screenshots
+* Copy Build ID to Clipboard
+* Copy List of Extensions to Clipboard
+* Copy about:support to Clipboard
 * Open Profile Folder
+* Open Pushlog
+* Screenshots utility
+* Titlebar customization
```
Comment (whimboo) [inline README.md]: Lets fix `Title bar` here too.

File: extension/chrome/content/aboutNightly/aboutNightly.xhtml [hunk 3]
```diff
@@ -44,11 +44,15 @@
...
-      <li>copy extensions list to clipboard</li>
-      <li>screenshot utility</li>
-      <li>open profile folder</li>
+      <li>Copy Build ID to Clipboard</li>
+      <li>Copy List of Extensions to Clipboard</li>
+      <li>Copy about:support to Clipboard</li>
+      <li>Open Profile Folder</li>
+      <li>Open Pushlog</li>
+      <li>Screenshots utility</li>
+      <li>Titlebar customization</li>
```
Comment (whimboo) [inline extension/chrome/content/aboutNightly/aboutNightly.xhtml]: and here.

**Comment by whimboo** [comment]
Only some nits left here. Otherwise it looks fine to me. Thanks.

File: History.md (hunk 1 above)
Comment (UtiluMark) [inline History.md]: @whimboo Do you want all the occurrences of "titlebar" changed to "title bar"? Also in the locales?

> History.md
> Line 14: * Extension: Make dtd entities for Customize Titlebar more localization friendly. (#140)
> Line 16: * Extension: Add basic Seamonkey support to titlebar customization. (#31)
> Line 34: * Extension: Make columns of 'Customize Titlebar' dialog's tree resizable. (#25)
> Line 41: * Extension: Open customize.xul (Customize Titlebar) as a resizable window. (#55)
> Line 43: * Extension: Include changeset info in the Insert/Copy Build ID menuitem and in 'Customize Titlebar'. (#65)
> Line 61: * Extension: Allow titlebar customization in Thunderbird (#24)
> Line 62: * Extension: Version 3.1 loses ability to customize titlebar with just window title (#18)

> \extension\chrome\locale\en-US\customize.dtd
> Line 10: <!ENTITY nightly.customize.title              "Customize Titlebar">
> Line 12: <!ENTITY nightly.paneTitlebar.title           "Titlebar">
> Line 13: <!ENTITY nightly.paneTitlebar.description     "Titlebar Customization">

> \extension\chrome\locale\en-US\nightly.dtd
> Line 23: <!ENTITY nightly.customize.label              "Customize Titlebar">

Comment (whimboo) [inline History.md]: We don't want to change the entity names but it might be good to update the actual output. @xabolcs what do you think?

Comment (xabolcs) [inline History.md]: Sure, keep entity names.

Comment (UtiluMark) [inline History.md]: @whimboo @xabolcs Shall I replace "titlebar" with "title bar" everywhere except in the entity names?

Comment (whimboo) [inline History.md]: I would say yes, at least for en-US. Maybe we should not touch other locales.

**Comment by UtiluMark** [comment]
@whimboo I've replaced "titlebar" with "title bar" everywhere except in entity names and reverted the Copy about:support to Clipboard feature to Pastebin.
Please review and merge.

File: README.md [hunk 4]
```diff
@@ -37,4 +40,5 @@ All bugs are reported to the Nightly Tester Tools component at bugzilla.mozilla.
 This project uses [.editorconfig](http://editorconfig.org/#overview), which sets defaults for the formatting of the code. So enjoy the use of [compatible editor](http://editorconfig.org/#download). Just download and install the corresponding plugin.
 
 # Related Add-ons:
-* [Add-on Compatibility Reporter](https://addons.mozilla.org/en-US/firefox/addon/15003/)
+* [Screengrab (fix version)](https://addons.mozilla.org/en-US/firefox/addon/screengrab-fix-version/)
+* [Add-on Compatibility Reporter](https://addons.mozilla.org/en-US/firefox/addon/add-on-compatibility-reporter/)
```
Comment (whimboo) [inline README.md]: Lets keep the add-on compat reporter as first item so we have an alphabetical sorted list.

**Comment by whimboo** [comment]
Looks fine to me beside the nit. Once an update has been pushed and you (@xabolcs) are ok with the changes, can you please get it landed? Thanks.

**Comment by xabolcs** [comment]
> Once an update has been pushed and you (@xabolcs) are ok with the changes, can you please get it landed?

Sure!

File: README.md (hunk 4 above)
Comment (UtiluMark) [inline README.md]: Done.

**Comment by UtiluMark** [comment]
@xabolcs Update done, so you can get this landed.

**Comment by xabolcs** [comment]
@UtiluMark 
Sorry for last minute review, but what are the benefits preventing redirects of AMO links?

**Comment by whimboo** [comment]
FYI by default we squash all commits at the end and give the final commit a proper comment (please see git log). @UtiluMark maybe you can do it to make it easier for @xabolcs to get it landed.

**Comment by UtiluMark** [comment]
@xabolcs The benefits of preventing redirects of AMO links are that the current links have more readable URLs (for example "nightly-tester-tools" in the URL is more descriptive compared to the number "6543" of the old URL).
The old AMO links are currently redirected to the new ones using a 301 MOVED PERMANENTLY redirect, these redirects might get removed at some point in the future since the URLs changed permanently, so the old AMO links might end up in 404 FILE NOT FOUND errors in the future.

**Comment by xabolcs** [comment]
Thanks @UtiluMark!

**Comment by xabolcs** [comment]
Looks good to me.

Ohh, almost forgot. @whimboo could you provide a commit message for this PR? :)
Thanks!

**Comment by whimboo** [comment]
I would like to let @UtiluMark do the squashing of commits and the commit message as pointed out in my last comment. @UtiluMark do you think you can do it? If you need help with let us know. In general make use of `git rebase -i master` and fixup all commits except the first one which needs an updated message.

**Comment by xabolcs** [comment]
@whimboo it's OK to me if you would like to teach @UtiluMark contributing to NTT, but a squashed commit of this PR is already in my working copy. It's OK to me to do the squashing work. :)

**Comment by UtiluMark** [comment]
@xabolcs I've just done what @whimboo asked. If you already have a squashed commit thats fine. :)

**Comment by xabolcs** [comment]
@UtiluMark, a `git commit --amend` is missing for me, to update the commit message in my copy.

**Comment by UtiluMark** [comment]
@xabolcs Done, changed the commit message to "Updated and synced feature lists and updated redirect links".

**Comment by whimboo** [comment]
There are still 39 commits listed on this PR. You might not have pushed via -f. Also for comments we add the issue number in brackets at the end.

**Comment by UtiluMark** [comment]
So the commit message for this PR should be:
> Updated and synced feature lists and updated redirect links (#198)
@xabolcs You wrote you already have a squashed commit of this PR, so there is nothing more to be done for me, right?

**Comment by xabolcs** [comment]
@UtiluMark, sure, but @whimboo would like to let you do all the work. :)

The commit message is almost correct. It should refer to #188 for now.

**Comment by xabolcs** [comment]
@UtiluMark, take the squash & rebase exercises on the next PR! :)

Merging this.

**Comment by xabolcs** [comment]
Landed as commit f313b2b2f6fb950cb5ff6633897c91ceb47b6255.
